tests/epyccel/test_default_precision_template.py

```diff
@@ -0,0 +1,29 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring
+
+from numpy.random import randint
+from numpy import isclose
+import numpy as np
+
+from pyccel.decorators import types
+from pyccel.epyccel import epyccel
+
+
+def test_default_precision_template(language):
+
+    @types('int[:]')
+    @types('float[:]')
+    @types('complex[:]')
+    def return_array_element(array):
+        return array[0]
+
+    test_types = ['int', 'float', 'complex']
+    f1 = return_array_element
+    f2 = epyccel(f1, language=language)
+    for t in test_types:
+        d1 = randint(1, 15)
```

Comment:
You sure this test works for all types of complex numbers? I mean `2+3j`, `3j` or `2`?

Reply:
It seems like it's only using real part of the complex numbers that's why I asked.

Reply:
I'm not sure that matters as long as we are testing with a complex type, but i may be wrong.